app/views/register/confirm.html.haml

```diff
@@ -51,6 +51,6 @@
   %i.icon.icon-important
     %span.visually-hidden Warning
   %strong.bold-small
-    All updates will be published to the Country register
+    Once you submit, these changes will be reviewed by another team member
```

Comment:
This wording is not really appropriate for custodians/advanced users as they can potentially approve their own changes. I have raised this with Gurinder but I think we can pick this up as a separate change.